content/influxdb/cloud-dedicated/reference/influxql/_index.md

```diff
@@ -0,0 +1,733 @@
+It is specified using Go's date / time parsing format, which is a reference date written in the format required by InfluxQL.
+The reference date time is:
+
+InfluxQL reference date time: January 2nd, 2006 at 3:04:05 PM
+
+```
+time_lit            = "2006-01-02 15:04:05.999999" | "2006-01-02" .
+```
+
+#### Booleans
+
+```
+bool_lit            = TRUE | FALSE .
+```
+
+#### Regular Expressions
+
+```
+regex_lit           = "/" { unicode_char } "/" .
+```
+
+**Comparators:**
+`=~` matches against
+`!~` doesn't match against
+
+{{% note %}}
+**NOTE:** InfluxQL supports using regular expressions when specifying:
+* [field keys](/influxdb/v2.7/reference/glossary/#field-key) and [tag keys](/influxdb/v2.7/reference/glossary/#tag-key) in the [`SELECT` clause](/influxdb/v2.7/query-data/influxql/explore-data/select/)
+* [measurements](/influxdb/v2.7/reference/glossary/#measurement) in the [`FROM` clause](/influxdb/v2.7/query-data/influxql/explore-data/select/#from-clause)
+* [tag values](/influxdb/v2.7/reference/glossary/#tag-value) and string [field values](/influxdb/v2.7/reference/glossary/#field-value) in the [`WHERE` clause](/influxdb/v2.7/query-data/influxql/explore-data/where/)
+* [tag keys](/influxdb/v2.7/reference/glossary/#tag-key) in the [`GROUP BY` clause](/influxdb/v2.7/query-data/influxql/explore-data/group-by/)
+
+Currently, InfluxQL does not support using regular expressions to match non-string field values in the `WHERE` clause, [databases](/influxdb/v2.7/reference/glossary/#database), and [retention polices](/influxdb/v2.7/reference/glossary/#retention-policy-rp).
+{{% /note %}}
+
+## Queries
+
+A query is composed of one or more statements separated by a semicolon.
+
+```
+query               = statement { ";" statement } .
+
+statement           = explain_stmt |
+                      explain_analyze_stmt |
+                      select_stmt |
+                      show_databases_stmt |
+                      show_field_key_cardinality_stmt |
+                      show_field_keys_stmt |
+                      show_measurement_exact_cardinality_stmt |
+                      show_measurements_stmt |
+                      show_series_exact_cardinality_stmt |
+                      show_series_stmt |
+                      show_tag_key_cardinality_stmt |
+                      show_tag_key_exact_cardinality_stmt |
+                      show_tag_keys_stmt |
+                      show_tag_values_with_key = stmt |
+                      show_tag_values_cardinality_stmt .
+```
+
+## Statements
+
+### EXPLAIN
+
+Parses and plans the query, and then prints a summary of estimated costs.
+
+Many SQL engines use the `EXPLAIN` statement to show join order, join algorithms, and predicate and expression pushdown.
+Since InfluxQL does not support joins, the cost of a InfluxQL query is typically a function of the total series accessed, the number of iterator accesses to a TSM file, and the number of TSM blocks that need to be scanned.
```

Comment:
```suggestion
Since InfluxQL doesn't support joins, the cost of an InfluxQL query is typically a function of the total series accessed, the number of iterator accesses to a TSM file, and the number of TSM blocks that need to be scanned.
```